coalib/bearlib/naming_conventions/__init__.py

```diff
@@ -130,3 +130,45 @@ def to_spacecase(string):
     return re.sub('(?<=[^\s])([A-Z])',
                   lambda match: ' ' + match.group(1),
                   string)
+
+
+def to_kebabcase(string):
+    """
+    Converts the given string to kebab-case.
+
+    >>> to_kebabcase('HelloWorld')
+    'hello-world'
```

Comment:
` String ` would be `--string-` rather than `-string-` there which looks alright. I still feel like we should follow a certain uniform standard which npm would do well for across all the naming conventions, it'd make it easier for conversions between them two conventions as well if ever required.
This may as well be a separate issue if you feel it's unnecessary.